docs/docs/firmware/state-monitor-esp32.md

```diff
@@ -99,6 +99,22 @@ To configure input 4 to be a contact sensor, and input 7 to be an inverted butto
 A retained message will ensure the device auto-configures on startup.
 :::
 
+### Recommended Configurations
+Below is a table showing possible **Connected Devices** and the supported `input types`. Check marks are indicating the recommended combinations to ensure intended behavior.
+
+|Connected Devices|`button`|`contact`|`press`|`rotary`|`switch`|`toggle`| 
+|:----------------|:------:|:-------:|:-----:|:------:|:------:|:------:|
+|**bi-stable Switches**   |:x:|:x:|:x:|:x:|:white_check_mark:|:white_check_mark:|
+|**Door / Window Contact**|:x:|:white_check_mark:|:x:|:x:|:x:|:x:|
+|**PIR**                  |:x:|:x:|:white_check_mark:|:x:|:x:|:x:|

```

Comment:
PIR should be recommended as `contact`

Reply:
isn't both `contact `and `press `usefull for a PIR. They usually send a short signal.

Reply:
You generally want to know when a PIR opens and closes - i.e. indicates motion and motion-stopped. It is a security sensor and as such is typically modelled as a contact. I think it is confusing to say `press` is a recommended config. Sure it would work but `press` doesn't really make sense with a PIR I don't think.